tests/helpers/client_side_db.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/python
+
+import logging
+import os
+import tempfile
+import getpass
+import requests
+import srp._pysrp as srp
+import binascii
+import json
+
+from twisted.internet.defer import inlineCallbacks
+
+from leap.soledad.client import Soledad
+
+
+"""
+Script to give access to client-side Soledad database.
+
+This is mainly used for tests, but can also be used to recover data from a
+Soledad database (public/private keys, export documents, etc).
+
+To speed up testing/debugging, this script can dump the auth data after
+logging in. Use the --export-auth-data option to export auth data to a file.
+The contents of the file is a json dictionary containing the uuid, server_url,
+cert_file and token, which is enough info to instantiate a soledad client
+without having to interact with the webapp again. Use the --use-auth-data
+option to use the auth data stored in a file.
+
+Use the --help option to see available options.
+"""
```

Comment:
As far as I can see in the code this is not a script meant to run standalone, but a set of functions to be used from scripts outside. Could you update or remove this comment, so we are not misleaded?

It might be more clear if the function that are meant to be used outside this file don't start with `_` in their name.